.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+<!-- Thank you for your contribution!
+     
+     Please file this form by replacing markdown commentary
+     tags with the text. If section needs in no action - remove it.
+
+     Also remember, that CouchDB uses the Review-Then-Commit (RTC) model 
+     of code collaboration. Positive feedback provides by +1 from committers
+     while negative by -1. The -1 also means veto and need to be addressed
+     to find the consensus. Once there are no objections, PR could be merged.
+
+     See: http://couchdb.apache.org/bylaws.html#decisions for more info -->
+
+## Overview
+
+<!-- Please give a short brief for the pull request,
+     what problem it solves or how it makes things better. -->
+
+## Testing recommendations
+
+<!-- Describe how we can test your changes. -->
+
+## Screenshot
+
+<!-- If visual or design changes were made, please provide the screenshots
+     of how it was and how it looks now in the way to easy detect the positive
+     improvements. -->
+
+## JIRA issue number
```

Comment:
Yes it is. Just waiting for this ticket https://issues.apache.org/jira/browse/INFRA-13627 for it to be enabled.